.travis.yml

```diff
@@ -0,0 +1,74 @@
+language: cpp
+dist: xenial
+git:
+  depth: 5
+  quiet: true
+
+python: 3.7
+
+
+
+matrix:
+  include:
+  # todo
+    # - name: Static checks (clang-format)
```

Comment:
I would suggest to leave out the commented out part and issue a separate PR where this all is implemented.